package work.one;

import java.sql.Connection;
import java.sql.PreparedStatement;
import java.sql.SQLException;

import jdbc.member.exam.DBUtil;

public class CustomerDAO {
	public int insert(CustomerDTO customer) {
		Connection con = null;
		PreparedStatement ptmt = null;
		int result = 0;
		String sql ="insert into CUSTOMER values(?,?,?,?,?,?)";
		try {
			con = DBUtil.getConnect();
			ptmt = con.prepareStatement(sql);
			ptmt.setString(1, customer.getId());
			ptmt.setInt(2, customer.getPass());
			ptmt.setString(3, customer.getName());
			ptmt.setDate(4, customer.getRegdate());
			ptmt.setInt(5, customer.getPoint());
			ptmt.setString(6, customer.getAddr());
			result = ptmt.executeUpdate(); 
		}catch(SQLException e) {
			e.printStackTrace();
		}finally {
			DBUtil.close(null, ptmt, con);
		}
		return result;
	}
}
